content/en/network_performance_monitoring/guide/manage_traffic_costs_with_npm.md

```diff
@@ -0,0 +1,64 @@
+or monitor your own team’s output specifically.
+{{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/region_region.png" alt="Use the region tag.">}}
+
+4. To monitor costs from external traffic, scope your destination endpoints to public IPs using the IP Type facet.
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/scope_destination_points.png" alt="Use the type facet.">}}
+    Then group your destination by `dns` to break down external traffic volume by where it’s going. Although you cannot install an agent on public servers, we are able to resolve external endpoints to human-readable domain names using DNS resolution. 
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/dns_resolution.png" alt="Group by DNS.">}}
+    The example query above filters for traffic to AWS S3, elastic load balancers, APIs, and external ‘.com’ domains using substring wildcard entries (e.g. dns:*s3*).  
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/wildcard.png" alt="Search with wildcards">}}
+
+## Visualizing Traffic Costs 
+
+You can visualize cross-AZ or inter-AZ traffic using the Network Map to quickly pinpoint bottlenecks. At Datadog, we also use this view to validate that our EU and US availability zones are not communicating, for compliance reasons. 
+(Cross-AZ traffic)
+{{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/cross-az-traffic.png" alt="Cross-AZ traffic">}}
+(Inter-AZ service-to-service traffic) 
+{{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/inter-az-service-to-service-traffic.png" alt="Inter-AZ service-to-service traffic">}}
+By default, the thickness of the gray lines between the availability zones indicates the volume of traffic flowing between them, which is what contributes to your costs.
+
+You can edit your preferences using the ‘Filter traffic’ button. In larger environments, we recommend scoping to just the most significant traffic sources by moving the sliders to include just the highest-volume dependencies.
+
+{{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/filter-traffic.png" alt="Scope your traffic">}}
+
+## Graphing Traffic Costs 
+
+We recommend tracking traffic volume metrics over time in dashboards and notebooks. You can graph traffic between any two endpoints using the same queries you would make in the network page. Just create a Timeseries Widget and select the ‘Network Traffic’ source from the dropdown menu.  
+
+{{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/timeseries.png" alt="Create a Timeseries">}}
+
+You can also use this data as the source of a monitor - so you can get a notification if traffic volume increases unexpectedly. This can give you a heads up that there’s something unusual happening before you get your cloud bill at the end of the month.
```

Comment:
We don't have monitors yet! So let's remove this sentence, and replace with something about how dashboards and notebooks make it easy for you to share any issues you spot with your teammates.